SearchBar/index.js

```diff
@@ -140,7 +142,7 @@ export default class SearchBar extends React.Component {
                             ref="filterBox"
                             data={data}
                             onChange={this.onChange}
-                            autoCompleteHandler={this.autoCompleteHandler}
+                            autoCompleteHandler={this.autoCompleteHandler()}
```

Comment:
this will create a new AutoCompleteHandler for every re-render.
instead maintain `data`, `options`, `scroll`, and `autoCompleteHandler` in component state. introduce `getDerivedStateFromProps()` and update `autoCompleteHandler` in state if `data` or `options` or `scroll` changes